src/components/transactions/TxDetails/TxData/DecodedData/index.tsx

```diff
@@ -49,7 +47,7 @@ export const DecodedData = ({ txData, toInfo }: Props): ReactElement | null => {
     ? 'this Safe Account'
     : addressInfo?.name || toInfo?.name || txData.to.name
   const avatar = addressInfo?.logoUri || toInfo?.logoUri || txData.to.logoUri
-  const isFallback = !method && !txData?.dataDecoded?.parameters
+  const isFallback = !method || txData?.dataDecoded?.parameters?.length === 0
```

Comment:
This will cover the case listed where there is a method but no parameters. I am not sure if we have the case where there is no method but parameters (don't think so) but in that case it would also fallback to raw data now.

Reply:
There are plenty of methods with no parameters.
E.g. WETH wrap/unwrap (value is passed as `value`, not in parameters), NFT mints etc.

<img width="590" alt="Screenshot 2024-12-06 at 15 07 54" src="https://github.com/user-attachments/assets/a6a3ec6a-2289-45fc-a6e9-c6a047946202">

Reply:
That case is fine and should now display the raw data instead of "No parameters"
<img width="1258" alt="Screenshot 2024-12-06 at 15 20 09" src="https://github.com/user-attachments/assets/a38e1bbb-40d2-4639-95a9-c7a5419fd3cc">

I was talking about the opposite case where potential transactions don't have a method name but parameters for some reason but feels like if the method name is missing then surely the ABI will be missing so there won't be decoded parameters either.

Reply:
I wouldn't replace "No parameters" with a raw data hash. The raw data could be added to the list below.

Reply:
Ah I undertand what you mean now. There are legitimate transactions without parameters so we should clearly tell the user in those cases that there are no parameters but still have the fallback to raw data. Lets display both to them.

Reply:
Like so
<img width="522" alt="Screenshot 2024-12-06 at 15 31 25" src="https://github.com/user-attachments/assets/c78aa69a-8a06-4480-bc02-f95871adaaaa">

Reply:
Yes, exactly. And I would move it down, below the separator. It's a bit weird that the copy button would be in front though.

Reply:
The part below the separator is the `Summary` which we don't render for Multisends. I checked if we can just add another divider between but imo it looks strange:
<img width="850" alt="Screenshot 2024-12-09 at 10 17 19" src="https://github.com/user-attachments/assets/bfcb0d2f-7e72-46a4-a2fc-f726feda20f0">
<img width="494" alt="Screenshot 2024-12-09 at 10 17 29" src="https://github.com/user-attachments/assets/39addf94-95f4-483d-a9a9-7f2694369a22">

I would keep it as is since it has a very limited scope (only rendered when we also render "No parameters")